k8s-install/scripts/install_cni_test.go

```diff
@@ -61,15 +61,21 @@ func cleanup() {
 	if err != nil {
 		Fail("Could not get CWD")
 	}
-	out, err := exec.Command("docker", "run", "--rm", "--name", "cni_cleanup",
+
+	// Assemble our arguments.
+	args := []string{
+		"run", "--rm", "--name", "cni_cleanup",
 		"-e", "SLEEP=false",
 		"-e", "KUBERNETES_SERVICE_HOST=127.0.0.1",
 		"-e", "KUBERNETES_SERVICE_PORT=8080",
-		"-v", cwd+"/tmp/bin:/host/opt/cni/bin",
-		"-v", cwd+"/tmp/net.d:/host/etc/cni/net.d",
-		"-v", cwd+"/tmp/serviceaccount:/var/run/secrets/kubernetes.io/serviceaccount",
+		"-v", cwd + "/tmp/bin:/host/opt/cni/bin",
+		"-v", cwd + "/tmp/net.d:/host/etc/cni/net.d",
+		"-v", cwd + "/tmp/serviceaccount:/var/run/secrets/kubernetes.io/serviceaccount",
 		fmt.Sprintf("%s", os.Getenv("CONTAINER_NAME")),
-		"sh", "-c", "rm -rf /host/opt/cni/bin/* /host/etc/cni/net.d/*").CombinedOutput()
+		"sh", "-c", "rm -rf /host/opt/cni/bin/* /host/etc/cni/net.d/*",
```

Comment:
Did this to follow the pattern with `runCniContainer()` function above this one. It made debugging easier because I could just print out the full `docker run ...` statement and manually execute it, etc.